src/Components/Programs/Timeline.js

```diff
@@ -58,14 +58,14 @@ function Timeline() {
                                     color:item.color,
                                     borderColor:item.color,
                                     flex:1,
-                                    left:item.date[0][2]*10,
+                                    left:item.date[2][1]*5,
                                     borderRadius: 50,
                                     zIndex:1,
                                     }}
                                 >
                                     {item.event}
                                 </Event>
-                                <Stroke style={{width:(item.date[0][1]+5)*20}}></Stroke>
+                                <Stroke style={{width:(item.date[2][1]+5)*50}}></Stroke>
```

Comment:
Why is 'date[2][1]' taken here ?

Reply:
because the date[2][1] value is nearly same for both the events GSoC and rail girls but date[0][1] value is not same for both